index.html

```diff
@@ -25,7 +26,174 @@ <h1>If These Lands Could Talk</h1>
 
 		<!-- Footer -->
 
+		<footer>
+			<div class="wrapper">
+				<div class="footer-main">
+					<div class="footer-logo-btn">
+						<!-- logo -->
+						<a href="./index.html">
+							<img
+								src="./assets/icons/global/itlct-logo.svg"
+								alt="If These Lands Could Talk Logo"
+							/>
+						</a>
+
+						<!-- Button for modal -->
+						<button class="btn-la">Land Acknowledgement</button>
+					</div>
+					<!-- Links list -->
+					<!-- need to replace with the real page links -->
+					<ul>
+						<li>
+							<a href="./index.html">Home</a>
+						</li>
+						<li>
+							<!-- Need to double check once Our Impact team is started -->
+							<a href="#ourimpact.html">Our Impact</a>
```

Comment:
The href values for these values should use a `/` instead of the `#`

`href="#ourimpact.html"` -> `href=""/ourimpact.html"`
`href="./index.html"` -> `href="/index.html"`
etc.

It forms a url, so we want to copy the pattern you see in your browser's address bar

Reply:
Since there's an about.html file, you can test that one.

Reply:
Linking to Relative URLs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#linking_to_relative_urls